crates/turborepo-ui/src/tui/pane.rs

```diff
@@ -7,50 +7,40 @@ use ratatui::{
 };
 use tracing::debug;
 use tui_term::widget::PseudoTerminal;
-use turborepo_vt100 as vt100;
 
-use super::{app::Direction, Error};
+use super::{app::Direction, Error, TerminalOutput};
 
 const FOOTER_TEXT_ACTIVE: &str = "Press`Ctrl-Z` to stop interacting.";
 const FOOTER_TEXT_INACTIVE: &str = "Press `Enter` to interact.";
 
-pub struct TerminalPane<W> {
-    tasks: BTreeMap<String, TerminalOutput<W>>,
-    displayed: Option<String>,
+pub struct TerminalPane<'a, W> {
+    displayed_task: &'a String,
     rows: u16,
     cols: u16,
     highlight: bool,
+    tasks: &'a mut BTreeMap<String, TerminalOutput<W>>,
 }
 
-struct TerminalOutput<W> {
-    rows: u16,
-    cols: u16,
-    parser: vt100::Parser,
-    stdin: Option<W>,
-    status: Option<String>,
-}
-
-impl<W> TerminalPane<W> {
-    pub fn new(rows: u16, cols: u16, tasks: impl IntoIterator<Item = String>) -> Self {
+impl<'a, W> TerminalPane<'a, W> {
+    pub fn new(
+        rows: u16,
+        cols: u16,
```

Comment:
We can drop the rows,cols from this constructor. The resize logic is currently unused and that's the only place these are used (along with the getter `term_size` which is also unused). 

I'm in favor for canning them so we don't have a false sense that they're important/used in their current state.